packages/odata-service-inquirer/src/prompts/edmx/entity-helper.ts

```diff
@@ -9,6 +9,11 @@ import LoggerHelper from '../logger-helper';
 export type EntityAnswer = {
     entitySetName: string;
     entitySetType: string;
+    /**
+     * The name of the navigation property (e.g., 'Set') linking a parameterised entity set to its target entity set.
```

Comment:
It correct to refer to this as `the navigation property` its confusing as we already define a navigation property. This is the main entity parameter.